Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ocaml/opam:debian-10-ocaml-4.11
+
+LABEL maintainer="gracewgao"
+
+SHELL ["/bin/bash", "-c"]
+
+# updates environment variables
+ENV PYRE_BINARY='/home/opam/pyre-check/source/_build/default/main.exe'
+ENV PYRE_TYPESHED='/home/opam/pyre-check/stubs/typeshed/typeshed-master'
+ENV PYTHONPATH='/home/opam/pyre-check:$PYTHONPATH'
+ENV PATH='/home/opam/.opam/4.10.2/bin:/home/opam/pyre-check/scripts:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH'
```

Comment:
Wow, are `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` all really missing from the `PATH` by default?

Reply:
I'm not sure but it seemed unable to find programs like make etc. unless I did this? I think it's maybe because the $PATH variable isn't evaluating properly

Reply:
@gracewgao I think the problem is that you're using single quotes, not double quotes. You need double quotes for string interpolation to occur.